src/app/config/groups/thunks.js

```diff
@@ -82,9 +83,22 @@ export const fetchGroupsRequest = isNewSignIn => dispatch => {
               })
               .catch(error => {
                   dispatch(actions.loadGroupsFailure());
-                  //TODO: map responses to user friendly by content designer
-                  if (error) {
-                      notifications.add({ type: "warning", message: error?.message || error.status, autoDismiss: 5000 });
+                  if (error.status != null && error.status === 400) {
```

Comment:
Is the `!= null` instead of `!== null` deliberate?

Reply:
Hi, that was used as the backend wasn't ready yet and the errors object were coming inconsistent. will  check if we still need it. The whole errors display will be revisited as not all messages and statuses are designed yet. this is just high level, first iteration. .